wire-ios/Wire-iOS Tests/ConversationList/Container/ConversationListViewControllerTests.swift

```diff
@@ -49,7 +49,12 @@ final class ConversationListViewControllerTests: XCTestCase {
         mockIsSelfUserE2EICertifiedUseCase = .init()
         mockIsSelfUserE2EICertifiedUseCase.invoke_MockValue = false
 
-        let selfUser = MockUserType.createSelfUser(name: "Johannes Chrysostomus Wolfgangus Theophilus Mozart", inTeam: UUID())
+        let modelHelper = ModelHelper()
```

Comment:
Maybe just create one model helper in the setup and reuse it in the tests.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/4c77d25c238ecb8182cd121cff9e7b91950e98d9